frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDetails.tsx

```diff
@@ -69,7 +69,7 @@ const PipelineRunDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath,
   );
 
   const loaded = runLoaded && (versionLoaded || !!run?.pipeline_spec);
```

Comment:
Change the loading condition to runLoaded only and you can check the version error in another variable.
```suggestion
  const loaded = runLoaded;
  const isVersionError = !versionLoaded || !run?.pipeline_spec
```